apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -298,6 +524,113 @@ expression: "&schema"
         }
       },
       "additionalProperties": false
+    },
+    "SpaceportConfig": {
+      "type": "object",
+      "required": [
+        "external"
+      ],
+      "properties": {
+        "collector": {
+          "default": "https://127.0.0.1:50051",
+          "type": "string"
+        },
+        "external": {
+          "type": "boolean"
+        },
+        "listener": {
+          "default": "127.0.0.1:50051",
```

Comment:
And spaceport?

Reply:
very good question, checking

Reply:
a676ff45 got the root cause and fixed the test!